packages/toast/README.md

```diff
@@ -21,90 +21,59 @@ npm install @leafygreen-ui/toast
 ## Example
 
 ```js
-import Toast from '@leafygreen-ui/toast';
+import Button from '@leafygreen-ui/button';
+import { ToastProvider, useToast, Variant } from '@leafygreen-ui/toast';
 
-const [toastOpen, setToastOpen] = useState(true);
+const { pushToast, clearStack, getStack, updateToast } = useToast();
+const stack = getStack();
+
+const createToast = () => {
+  pushToast({
+    title: 'Toast Title',
+    description: 'Toast Description',
+    variant: Variant.Success,
+    timeout: null,
+  });
+};
 
 return (
-  <Toast
-    variant="success"
-    title="This is a title"
-    body="This is a description"
-    open={toastOpen}
-    close={() => setToastOpen(false)}
-  />
+  <ToastProvider>
+    <Button onClick={handleClick}>Create toast</Button>
```

Comment:
Is the click handler supposed to be `createToast`?

Reply:
if you read the click handler it does push a toast to the stack!

Reply:
Updated!